src/components/Login/login.jsx

```diff
@@ -0,0 +1,138 @@
+       await this.props.login(userLogin, password);
+       const { error, history: { push } } = this.props;
+       this.setState({ errors: { server: error } });
+       if (!error) push('/dashboard');
+     }
+   }
+
+   render() {
+     const {
+       state: {
+         userLogin, password, errors
+       },
+       props: { error, isLoading }
+     } = this;
+
+     return (
+       <>
+         <main className="Login">
+           <section className="Login__info">
+             <h2 className="Login__info__header">
+              Inspire the next generation of users by
+               <br />
+              continuously voicing your innovative ideas
+             </h2>
+             <p className="Login__info__text">Join the world&apos;s largest community of writers and readers</p>
+             <p className="Login__info__text">20M+ active users</p>
+             <p className="Login__info__text">Over 2000+ new stories daily</p>
+             <p className="Login__info__text">No hidden charges. Get started for free now!</p>
+           </section>
+           <div className="Login__form">
+             <p className="Login__form__header">Log In</p>
+             <Form className="form" onSubmit={this.handleSubmit}>
+               <Input
+                 type="text"
+                 className={errors.userLogin && 'has-validation-error'}
+                 error={errors.userLogin}
+                 name="userLogin"
+                 value={userLogin}
+                 placeholder="email or username"
+                 onChange={this.handleChange}
+                 onFocus={this.handleFocus}
+               />
+
+               <Input
+                 type="password"
+                 className={errors.password && 'has-validation-error'}
+                 error={errors.password}
+                 name="password"
+                 value={password}
+                 placeholder="password"
+                 onChange={this.handleChange}
+                 onFocus={this.handleFocus}
+               />
+               {error && (<p className="center input-validation-error">{errors.server}</p>)}
+               <div className="form-field">
+                 <Button
+                   className="btn"
+                   title="Please fill up the form"
+                   label={isLoading ? 'Please wait...' : 'Submit'}
+                   disabled={isLoading}
+                 />
+               </div>
+               <p className="center">Forgot Password?</p>
+             </Form>
+             <div className="form__footer">
+               <span>
+                Continue with:
+                 <img className="facebook-icon" src={FacebookIcon} alt="facebook" />
+                 <img className="google-icon" src={GoogleIcon} alt="google" />
+                 <img className="twitter-icon" src={TwitterIcon} alt="twitter" />
+               </span>
+               <span>Don't have an account? login</span>
```

Comment:
@jhayX I believe we expect this link should redirect users to the login page, right? You could make use of the `Link` component of `react-router-dom` package to achieve this.

Reply:
@jhayX In addition, Why should you redirect a user to login form again? Please, let's change this to signup.